src/requested-data/requested-data-actions.ts

```diff
@@ -97,10 +83,23 @@ export function initializeRequestedMetrics(metrics: Metrics[]) {
  *
  * @param parameters - Parameters fetched from the API.
  */
-export function InitializeRequestedParameters(parameters: Parameters[]) {
+export function initializeRequestedParameters(parameters: Parameters[]) {
     const action: InitializeRequestedParameters = {
         type: RequestedDataActionType.INIITALIZE_REQUESTED_PARAMETERS,
         parameters,
     };
     return action;
 }
+
+/**
+ * Action creator for initializing requested PartyMap.
+ *
+ * @param partyMap - PartyMap fetched from the API.
+ */
+export function initializeRequestedPartyMap(partyMap: Dictionary<string>) {
+    const action: InitializeRequestedPartyMap = {
+        type: RequestedDataActionType.INIITALIZE_REQUESTED_PARTY_MAP,
```

Comment:
These typos seem to repeat themselves, we should make a decision of which PR we fix them in. Maybe just leave them until we've merged things into develop and do a quick cleanup.

Reply:
Yup